src/test/groovy/steps/DataAnalyticsSteps.groovy

```diff
@@ -48,7 +42,7 @@ class DataAnalyticsSteps {
             def usernameRt = "user${count}"
             def emailRt = "email+${count}@server.com"
             def passwordRt = "password"
-            Response http201 = securitySteps.createUser(artifactoryBaseURL, username, password, usernameRt, emailRt, passwordRt)
+            Response http201 = securitySteps.createUser(artifactoryURL, username, password, usernameRt, emailRt, passwordRt)
```

Comment:
better curry example here:

```groovy
def mkuser = (securitySteps.&createUser).curry(artifactoryUrl,username,password)
mkuser(usernameRt,emailRt)
```
Anywhere you have these functions you really don't want to pass the same value over-and-over; following the `DRY` principle

Reply:
Done